gui/services/forms.py

```diff
@@ -1178,36 +1189,28 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super(iSCSITargetPortalForm, self).__init__(*args, **kwargs)
-        self.fields["iscsi_target_portal_tag"].initial = (
-            models.iSCSITargetPortal.objects.all().count() + 1)
+        self._listen = []
         self.fields['iscsi_target_portal_discoveryauthgroup'].required = False
-        self.fields['iscsi_target_portal_discoveryauthgroup'].choices = [('-1', _('None'))] + [(i['iscsi_target_auth_tag'], i['iscsi_target_auth_tag']) for i in models.iSCSITargetAuthCredential.objects.all().values('iscsi_target_auth_tag').distinct()]
+        self.fields['iscsi_target_portal_discoveryauthgroup'].choices = [(None, _('None'))] + [(i['iscsi_target_auth_tag'], i['iscsi_target_auth_tag']) for i in models.iSCSITargetAuthCredential.objects.all().values('iscsi_target_auth_tag').distinct()]
```

Comment:
Why not move it to middleware plugin? New UI will need this list too.

Reply:
@themylogin that list is already available via API 1.0 and will be in websocket as well once `iscsi.auth` is implemented.